starter-code/css/main.css

```diff
@@ -0,0 +1,98 @@
+* {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+}
+
+body {
+    width: 900px;
+    margin: 0 auto;
+}
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    font-size: 65px;
+    padding: 0px;
+    margin: 10px 0px;

```

Comment:
Whenever you are using 0 as a value you do not need to specify the units. 0px = 0% = 0.